README.md

```diff
@@ -40,21 +41,47 @@
 
 
 
-## 📖 Math Magician <a name="about-project"></a>
+## 📖  book-store <a name="about-project"></a>
 
 **[Book-store]** is a basic website that allows users to add/remove books from a list.
 
 ## 🛠 Built With  VSCode  <a name="Built With VSCode"></a>
 
-- React
-- Javascript
-- Html5
-- Css3
-
-### Tech Stack <a name="tech-stack"></a>
-
-### HTML/CSS, javascript, Webpack <a name="tech-stack"></a>
-
+### HTML/CSS, javascript, Webpack, React and Redux <a name="tech-stack"></a>
+
+>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Styling</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Functionality</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Redux</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+  </ul>
+</details>
```

Comment:
- [ ] I see that you added the technologies you used, however, please make sure that you have the correct names (a lot of them are repeated) and links. Also, please include a "Key features" section as suggested by the previous reviewer:

![image](https://user-images.githubusercontent.com/77069011/236953277-4051e955-82c2-4a2a-83c0-efb7d2400fc4.png)

Please consider making this change! 🤓 🔧